ir/instr.cpp

```diff
@@ -872,6 +873,11 @@ StateValue UnaryOp::toSMT(State &s) const {
       return v.cttz(expr::mkInt(-1, v)) + expr::mkUInt(1, v);
     };
     break;
+  case FAbs:
```

Comment:
Looking at FNeg handling above, doesn't this need fast-math flags?

Reply:
You're right, thanks..! Forgot that call instructions can take fast-math flags.